component/core/src/main/java/org/exoplatform/social/core/jpa/storage/RDBMSIdentityStorageImpl.java

```diff
@@ -1107,6 +1143,15 @@ public ActivityStorage getActivityStorage() {
     return activityStorage;
   }
 
+  private static String getNameAbbreviation(String name) {
```

Comment:
To move in the bottom of the file after the `public` methods (Code conventions, public methods should be declared before private ones)
Besides, no need for the `static` modifer

```suggestion
  private String getNameAbbreviation(String name) {
```